editor.html

```diff
@@ -341,14 +341,23 @@ <h2 id="flashing-text">Flashing micro:bit</h2>
         </div>
         <div id="modal-msg-overlay-container" class="modal-overlay-container modal-msg-overlay-container">
           <div id="modal-msg-overlay" class="modal-overlay">
-            <a class="vex-close" id="modal-msg-close" onclick="$('#modal-msg-overlay-container').hide()"></a>
+            <a class="vex-close modal-close" onclick="$('#modal-msg-overlay-container').hide()"></a>
             <div>
               <h2 id="modal-msg-title"></h2>
               <div id="modal-msg-content"></div>
               <div id="modal-msg-links" class="modal-msg-links"></div>
             </div>
           </div>
         </div>
+        <div id="filetype-overlay-container" class="modal-overlay-container">
```

Comment:
We already have the modal-msg, which can be triggered with the `modalMsg()` function you created, so we can use that instead of defining a new modal :)

Reply:
Got it, done that and had to edit modal-msg to make it suitable. Do you want to change the appearance of these at all? 
<img width="1036" alt="Screenshot 2019-09-26 09 30 30" src="https://user-images.githubusercontent.com/53710678/65679702-87629800-e04d-11e9-987e-03743eee9f72.png">
<img width="651" alt="Screenshot 2019-09-26 09 30 41" src="https://user-images.githubusercontent.com/53710678/65679719-8fbad300-e04d-11e9-95d7-a18b7f63df5f.png">
<img width="588" alt="Screenshot 2019-09-26 09 30 54" src="https://user-images.githubusercontent.com/53710678/65679739-9a756800-e04d-11e9-833c-de06815413d5.png">

Reply:
Looks good to me! Did you test them in all browsers?

Reply:
If you are using `modalMsg()` now, should this `filetype-overlay-container` div be removed?